src/server/status_server/mod.rs

```diff
@@ -677,6 +783,9 @@ where
                             (Method::GET, "/debug/pprof/heap") => {
                                 Self::dump_heap_prof_to_resp(req).await
                             }
+                            (Method::GET, "/debug/pprof/cmdline") => Self::get_cmdline(req).await,
+                            (Method::GET, "/debug/pprof/symbol") => Self::get_symbol(req).await,
+                            (Method::POST, "/debug/pprof/symbol") => Self::get_symbol(req).await,
```

Comment:
why do we need to support both post and get for symbol, with same implementation.

Reply:
OK, I saw the implementation of get_symbol is dependent on method. I think it's better to separate it for get and post respectively.